src/main.rs

```diff
@@ -183,8 +187,8 @@ fn main() -> Result<(), Box<dyn Error>> {
 /// introduction to the meaning of the configured `options`, to `stdout`.
 fn usage(options: &getopts::Options) {
     println!(
-        "Parses the output of `git log`.\n\n{}",
-        options.usage("Usage: commit-analyzer <FILE> [OPTIONS]")
+        "Parses the Git history.\n\n{}",
+        options.usage("Usage: commit-analyzer [OPTIONS]")
```

Comment:
The in-app docs needed to be adjusted.  Suggestions for better wordings are welcome.  :wrench:

Reply:
Hmm, why though? Maybe we need to be more precise, how that output can be supplied, but at its core this is still true.

Reply:
"Grabs and parses the output of `git log --numstat`." :wrench: :question:

Reply:
I am actually not sure, If the in-app docs need to be adjusted. I just tried it and ended up with the original text.. 😅 

The problem with yours is, that is only true if something like `--git` would be supplied.

Reply:
:wrench: :heavy_check_mark:

Reply:
Did you mean to change this?

Reply:
Yes, I meant to change it.  And I already changed it:  you now always an option to specify the input.  So the synopsis part is correct.

Reply:
The sole open point might be that the description should receive a change.  If so, please provide a suggestion. :wrench:

Reply:
I think "Parses the Git history" is quite colloquial and maybe not even correct. I have never heard git commits being called the git history. If you have something, that does call it that, I'd be interested. 😄